icubam/www/handlers/disclaimer.py

```diff
@@ -0,0 +1,29 @@
+import os.path
+
+from icubam.www.handlers import base
+
+
+class DisclaimerHandler(base.BaseHandler):
+  ROUTE = '/disclaimer'
+
+  def initialize(self, config, db_factory):
+    super().initialize(config, db_factory)
+
+  def get_disclaimer_html(self):
+    """To show a disclaimer page if specified in configuration."""
+    path = self.config.server.disclaimer
+    # The user has already agreed, we skip.
+    if path is not None and os.path.exists(path):
+      with open(path, 'r') as fp:
+        return fp.read()
```

Comment:
Instead of opening the file here, can we just `{% include *file_path* %}` in the template? I guess that might also use the template caching (not sure) https://www.tornadoweb.org/en/stable/guide/templates.html#configuring-templates

Reply:
Also if path is in the config but it not valid, I feel we should at least raise a warning to indicate that something is wrong.

Reply:
> Instead of opening the file here, can we just `{% include *file_path* %}` in the template? I guess that might also use the template caching (not sure) https://www.tornadoweb.org/en/stable/guide/templates.html#configuring-templates

If I use `{% include '%s' % disclaimer %}` with `disclaimer` as the file path, I am facing:
`FileNotFoundError: [Errno 2] No such file or directory: "/path/to/icubam/icubam/www/templates/%s' % disclaimer"`
This is because include is generally used to include a template in another one. This means I would have to add another template directory with the one where disclaimer.html is.
I think the `{% raw disclaimer %}` solution is better, no ? But I can understand that if the html is big it is not a nice solution.

Reply:
Fair enough. Thanks for checking.